core/settings.py

```
@@ -24,13 +24,25 @@
-
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.environ.get('DEBUG', 'True').lower() == 'true'
 
-ALLOWED_HOSTS = ['.vercel.app', '*']
+from django.core.exceptions import ImproperlyConfigured
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get('SECRET_KEY')
+if not SECRET_KEY:
+    if DEBUG:
+        SECRET_KEY = 'django-insecure-dev-key-for-local-testing'
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)